packages/gatsby/src/redux/actions/public.js

```diff
@@ -963,7 +963,7 @@ type CreateNodeInput = {
  * @param {string} $0.fieldName [deprecated] the name for the field
  * @param {string} $0.fieldValue [deprecated] the value for the field
  * @param {string} $0.name the name for the field
- * @param {string} $0.value the value for the field
+ * @param {any} $0.value the value for the field
```

Comment:
I think we actually sanitize it to this set of types:

https://github.com/gatsbyjs/gatsby/blob/2b578e81e529914778fe92aa24446e1b1b0f3949/packages/gatsby/src/db/sanitize-node.js#L68-L72

Not sure if we should change `any` to this union though

Reply:
On the other hand they can pass anything here indeed. It will be simply stripped away unless it's `null | number | string | boolean | Date`

Reply:
It can also be a plain object or array.